owncloudData/src/main/java/com/owncloud/android/data/storage/LegacyStorageProvider.kt

```diff
@@ -0,0 +1,30 @@
+/**
+ * ownCloud Android client application
+ *
```

Comment:
Who is the author? A ghost? 👻 . In `LocalStorageProvider.kt` we did wrote our names hehe